google/cloud/spanner/value.cc

```diff
@@ -274,6 +288,17 @@ StatusOr<std::string> Value::GetValue(std::string const&,
   return pv.string_value();
 }
 
+StatusOr<Value::Bytes> Value::GetValue(Value::Bytes const&,
+                                       google::protobuf::Value const& pv,
+                                       google::spanner::v1::Type const&) {
+  if (pv.kind_case() != google::protobuf::Value::kStringValue) {
+    return Status(StatusCode::kUnknown, "missing BYTES data");
```

Comment:
FYI: Not covered by test.

Reply:
Tested now.

Reply:
The new test looks like it hits the decode error below, but not this "missing" case.  Perhaps the other "missing"s are not hit either?

Reply:
I think other missing cases are tested, like in the test named GetBadBool. I missed this one for Bytes, but I just added a couple more tests for `Value::Bytes`, which I think brings up to matching the other value-type tests.